handlers/holiday-stop-api/src/main/scala/com/gu/holiday_stops/Handler.scala

```diff
@@ -461,7 +468,7 @@ object Handler extends Logging {
   )(req: ApiGatewayRequest, sfClient: SfClient): ApiResponse = {
     val sfClientGetWithParams = sfClient.wrapWith(JsonHttp.getWithParams)
     val lookupOpHolidayStopsOp = LookupByContactAndOptionalSubscriptionName(sfClientGetWithParams)
-    val updateRequestDetailOp = CancelHolidayStopRequestDetail(sfClient.wrapWith(JsonHttp.post))
+    val updateRequestDetailOp = CancelHolidayStopRequestDetail.doRequest(sfClient.wrapWith(JsonHttp.post)) _
```

Comment:
rename `apply` to `doRequest` to make it clearer.  And the underscore shows that there's more to do to call the method.  Another option would be to make it even more explicit (eta expand it) with something like
```suggestion
    def updateRequestDetailOp(something: SomeType) = CancelHolidayStopRequestDetail.doRequest(sfClient.wrapWith(JsonHttp.post))(something)
```